parm/config/config.vrfy

```diff
@@ -47,6 +51,9 @@ if [ $VRFYFITS = "YES" ]; then
         export PREPQFITSH="$fitdir/subfits_s4_slurm"
     elif [ $machine = "ORION" ]; then
         export PREPQFITSH="$fitdir/subfits_orion_netcdf"
+    elif [ $machine = "JET" ]; then
+        export PREPQFITSH="$fitdir/subfits_jet_netcdf"
+        export COMINsyn="/lfs4/HFIP/hwrf-data/hwrf-input/SYNDAT-PLUS"
```

Comment:
Should be able to delete this `COMINsyn` line and use the `COMINsyn` set in `config.base` via the paths in `jet.yaml`. I keep the Jet and Hera syndat folders updated from WCOSS2 prod periodically.

Reply:
I will update and run a new test case.